experimental/dds/tree/src/id-compressor/IdCompressor.ts

```diff
@@ -1777,6 +1739,18 @@ export class IdCompressor {
 			}
 		}
 
+		if (serializedLocalState !== undefined) {
```

Comment:
This is a forwards-compatible change but not backwards-compatible, right? 

A client before this change can serialize, and a client after this change will load it up no problem.
But a client before this change cannot load a serialized compressor from a client after this change.

But it doesn't actually matter because we don't have any customers using local state yet, so no need for a flag + rollout.

Reply:
No, I believe that it's not compatible in either direction since it is now required for correctness. However, as you pointed out, no one has made local state so we can sidestep making a new verison.

Reply:
If it is required for correctness, then why do we tolerate undefined here?

Reply:
We are tolerating undefined for serializedLocalState, not for the normalizer. The undefinedness of serializedLocalState depends on whether the IdCompressor was serialized with session state or not, and isn't related to the normalizer.

Reply:
Oh yea, I was thinking this was a new variable for some reason. This is correct.